lib/tizen_cache.dart

```diff
@@ -248,6 +252,17 @@ class TizenEngineArtifacts extends EngineCachedArtifact {
       } finally {
         status.stop();
       }
+      _makeFilesExecutable(artifactDir, operatingSystemUtils);
+    }
+  }
+
+  void _makeFilesExecutable(
+      Directory dir, OperatingSystemUtils operatingSystemUtils) {
+    operatingSystemUtils.chmod(dir, 'a+r,a+x');
```

Comment:
Do we need this line?

Reply:
Yes, `Directory.createSynk()` method is related to the `umask` settings of the system. In general, `umask` is set to give `+x` to the creating directory, this line is needed as a safety measure to make sure.